Sources/Kitura/Router.swift

```diff
@@ -72,10 +85,17 @@ public class Router {
         defaultEngineFileExtension = nil
     }
 
+    /// Register a templating engine. The templating engine will handle files in the `viewsPath`
+    /// that match the extension it supports.
+    ///
+    /// - Parameter templateEngine: The templating engineto register.
     public func add(templateEngine: TemplateEngine) {
         templateEngines[templateEngine.fileExtension] = templateEngine
     }
 
+    /// Render a template using a context
+    ///
+    /// - Parameter template:
```

Comment:
text is now:    
    /// Render a template using a context
    ///
    /// - Parameter template: The path to the template file to be rendered.
    /// - Parameter context: A Dictionary of variables to be used by the
    ///                     template engine while rendering the template.
    ///
    /// - Returns: The content generated by rendering the template.
    /// - Throws: Any error thrown by the Templating Engine when it fails to
    ///          render the template.